gspread-project/README.md

```diff
@@ -0,0 +1,7 @@
+# GSPREAD Project
```

Comment:
Good job adding a readme for your project, you should also include other details about the project. You can check online for readme.md templates and tailor it to your taste.